cli/cmd/top.go

```diff
@@ -642,16 +691,18 @@ func (t *topTable) adjustColumnWidths() {
 				t.columns[i].width = cellWidth
 			}
 		}
+
+		tablewidth = tablewidth +  t.columns[i].width + 10
```

Comment:
Where do the `+10` and `+15` values come from? Should these be based on the `columnSpacing` somehow?

Reply:
I tried to use `columnSpacing` initially, but somehow it wasn't getting the right width of the table. By printing out `columnSpacing`, I realised it's maximum value is only 2, same case with the variable `padding`. Finally by Hit and Trial I figured out that `10` works perfectly for our scenario. Hence I used the same. :)